### ISSUE TITLE
Add title attribute to iframe elements

### ISSUE THREAD

**Issue by yileifeng** [ramp4-pcar4/storylines-editor#492]
## Description

Cross referencing https://github.com/ramp4-pcar4/tcei-tmx-cwa-storylines/issues/115, add a `title` attribute to `iframe` elements to avoid WCAG violation.

From accessibility test in Storylines: 

> 8. WCAG Violation Code: H64
Message: Iframe element requires a non-empty title attribute that identifies the frame.
Context: `<iframe class="media-player" src="https://www.youtube.com/embed/4kTq-hnrMHI" height="500px" allowfullscreen=""></iframe>`
Selector: `#4-cumulative-effects-video > div > div:nth-child(2) > div > iframe`
[Get Help](https://www.w3.org/TR/WCAG20-TECHS/H64)
